tests/longhaul/base_block_test.go

```diff
@@ -0,0 +1,299 @@
+package longhaul
+
+import (
+	"strings"
+	"sync"
+	"testing"
+
+	"github.com/coreos/pkg/capnslog"
+	"github.com/rook/rook/tests/framework/installer"
+	"github.com/rook/rook/tests/framework/utils"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"time"
+)
+
+var (
+	logger                   = capnslog.NewPackageLogger("github.com/rook/rook", "longhaul")
+	defaultNamespace         = "default"
+	defaultLongHaulNamespace = "longhaul-test"
+)
+
+func setUpRookAndPoolInNamespace(t func() *testing.T, namespace string, storageClassName string, poolName string) (*utils.K8sHelper, *installer.InstallHelper) {
```

Comment:
a lot of these helpers in the longhaul tests seem duplicated from the integration tests. For example, all of the test yaml and starting of services seems like it would be the same

Reply:
if there are subtle differences, can we parameterize the difference? Still seems like a lot of duplication.